crates/next-core/js/src/dev/hmr-client.ts

```diff
@@ -362,46 +597,3 @@ function triggerUpdate(msg: ServerMessage) {
     location.reload();
   }
 }
-
-// Unlike ES chunks, CSS chunks cannot contain the logic to accept updates.
-// They must be reloaded here instead.
-function subscribeToInitialCssChunksUpdates(assetPrefix: string) {
-  const initialCssChunkLinks: NodeListOf<HTMLLinkElement> =
-    document.head.querySelectorAll(`link[rel="stylesheet"]`);
-
-  initialCssChunkLinks.forEach((link) => {
-    subscribeToCssChunkUpdates(assetPrefix, link);
-  });
-}
-
-export function subscribeToCssChunkUpdates(
-  assetPrefix: string,
-  link: HTMLLinkElement
-) {
-  const cssChunkPrefix = `${assetPrefix}/`;
-
-  const href = link.href;
-  if (href == null) {
-    return;
-  }
-
-  const { pathname, origin } = new URL(href);
-  if (origin !== location.origin || !pathname.startsWith(cssChunkPrefix)) {
-    return;
-  }
-
-  const chunkPath = pathname.slice(cssChunkPrefix.length);
-  subscribeToChunkUpdate(chunkPath, (update) => {
-    switch (update.type) {
-      case "restart": {
-        console.info(`Reloading CSS chunk \`${chunkPath}\``);
-        link.replaceWith(link);
-        break;
-      }
-      case "partial":
-        throw new Error(`partial CSS chunk updates are not supported`);
-      default:
-        throw new Error(`unknown update type \`${update}\``);
-    }
-  });
-}
```

Comment:
CSS updates are now handled at the chunk list level, using the `BACKEND.loadChunk/unloadChunk/reloadChunk` logic.